web/src/pages/Profile/profile.scss

```diff
@@ -0,0 +1,29 @@
+.profile-container{
```

Comment:
You should remove this, this directory is already pushed in `current` under a new name